### PR TITLE
🛁 - Cleaned up readme

### PR DESCRIPTION
**Pull request by arb**
Fixed 404s. Updated to reflect current and future code.
Updated examples. Removed CONTRIBUTING.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,146 +1,79 @@
...
-| --------------- | ------------------ |
-| <= 1.1.5        | <= 0.0.3           |
-| 1.2 - 2.0.0     | 0.0.4, 0.0.5       |
-| 2.0.0           | >= 0.1.0           |
+Fork, clone, dev, write or ammend tests and PR. All PRs go against "master". This project uses [prettier](https://github.com/prettier/prettier) on file commit, so don't worry about style as it'll just get rewritten when you commit your changes.
 
+## Version details
 
-## Quick Example ##
+The current version of stardog.js has been tested against version 5.0.0 of Stardog. You are encouraged to use this library if you are using version 5 or greater of Stardog. However, there is very little code that is version specific in Stardog.js. It is essentially just a convenience wrapper around `fetch`. It is very likely that many of the exposed methods will work on older versions of Stardog, but it is has not been tested.
```
Comment (tysoeh) [inline README.md]: We should be consistent on `Stardog.js` or `stardog.js`

File: README.md [hunk 2]
```diff
@@ -1,146 +1,79 @@
-Stardog.js
-==========
+# Stardog.js
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/complexible/stardog.js?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Universal Javascript fetch wrapper for communicating the with Stardog HTTP server.
```
Comment (tysoeh) [inline README.md]: "with the Stardog HTTP server" instead of "the with"

File: README.md [hunk 3]
```diff
@@ -1,146 +1,79 @@
...
+## What is it?
 
 This framework wraps all the functionality of a client for the Stardog DBMS, and provides access to a full set of functions such as executing SPARQL Queries, administration tasks on Stardog, and the use of the Reasoning API.
 
-All the implementation uses the HTTP protocol, since most of Stardog functionality is available using this protocol. For more information, go to the Stardog's [HTTP Programming](http://docs.stardog.com/http/) documentation.
+All the implementation uses the HTTP protocol, since most of Stardog functionality is available using this protocol. For more information, go to the Stardog's [HTTP Programming](http://www.stardog.com/docs/#_network_programming) documentation.
 
-The framework is currently supported for node.js and the browser, including test cases for both environments.
-You'll also need [npm](https://npmjs.org) and [bower](http://bower.io) to run the test cases and install the dependencies in node.js & the browser respectively.
+This is a universal library and as such can be used in both the browser and node.
```
Comment (tysoeh) [inline README.md]: Capitalize "Node" I think

**Comment by kendall** [comment]
bower? 👀

File: README.md [hunk 4]
```diff
@@ -1,146 +1,79 @@
...
+```
 
 ### Running Tests
 
-Run all the test cases in `test/spec`, all test cases must pass. Having the Stardog server running, execute the following commands:
-
-* In the browser
-
-    1. Load the test data using the provided script:
+Run all the test cases in `test/spec`, all test cases must pass. With the Stardog server running, execute the following commands:
```
Comment (tysoeh) [inline README.md]: "In order to contribute changes, all test cases must pass. With the Stardog server running, execute the following:"?

**Comment by arb** [comment]
@kendall we weren't planning on continuing to push new versions to bower as it's usage gets lower and lower every day.

**Pull request merged**